content/docs/guides/responsive_amp/responsive_design@id.md

```diff
@@ -0,0 +1,173 @@
+---
+$title: Membuat halaman AMP responsif
+---
+[TOC]
+
+## Pendahuluan
+
+Desain web yang responsif adalah tentang membuat halaman lancar yang merespons kebutuhan pengguna Anda—halaman yang sesuai dengan orientasi dan ukuran layar perangkat mereka.  Anda dapat membuatnya dengan mudah di AMP.  AMP mendukung semua kategori perangkat dan layar, serta menyediakan komponen bawaan yang responsif.
+
+Dalam panduan ini, kami akan menunjukkan cara menerapkan dasar-dasar komponen responsif tersebut di AMP dengan mudah:
+
+- [Mengontrol viewport](#mengontrol-viewport)
+- [Membuat tata letak responsif](#membuat-tata-letak-responsif)
+- [Mengubah skala media](#mengubah-skala-media-untuk-halaman)
+
+## Mengontrol viewport
+
+Guna mengoptimalkan halaman sehingga konten diubah skalanya dan menyesuaikan jendela browser untuk semua perangkat, Anda harus menentukan elemen viewport `meta`. Elemen viewport memberikan petunjuk ke browser tentang cara mengubah skala dan menyesuaikan ukuran area yang terlihat (viewport) dari halaman.
+
+Namun, nilai apa yang harus digunakan?  Di AMP, nilai tersebut sudah tersedia untuk Anda.  Sebagai bagian dari [markup yang diperlukan](/id/docs/reference/spec.html#required-markup) untuk halaman AMP, Anda harus menentukan viewport berikut:
+
+```html
+<meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+```
+
+Setelan tersebut adalah setelan viewport umum yang digunakan untuk situs responsif. Meskipun `initial-scale=1` tidak diperlukan untuk halaman AMP yang valid, komponen ini direkomendasikan karena akan menetapkan level zoom ke 1 saat halaman dimuat pertama kali.
+
+## Membuat tata letak responsif
+
+Dengan desain yang responsif, Anda dapat menggunakan kueri CSS [`@media`](https://developer.mozilla.org/en-US/Web/CSS/@media) guna menyesuaikan gaya halaman untuk berbagai dimensi layar, tanpa harus mengubah konten halaman.  Di AMP, Anda tetap dapat menggunakan kueri `@media` CSS yang sama. Selain itu, agar elemen AMP mendapakan kontrol yang lebih baik, Anda dapat menentukan atribut `media` pada elemen. Cara ini khususnya berguna saat Anda perlu menampilkan atau menyembunyikan elemen berdasarkan kueri media. Lihat bagian [Mengubah gambar responsif dari suatu gambar](#changing-the-art-direction-of-an-image) untuk mengetahui contoh penggunaan atribut `media`.
```

Comment:
Translate `#changing-the-art-direction-of-an-image` -> Should correspond to header within this doc.